src/GovUk.Education.ExploreEducationStatistics.Content.Model/User.cs

```diff
@@ -12,6 +13,10 @@ public class User
 
         public string Email { get; set; }
 
+        public Guid DeletedById { get; set; }
```

Comment:
I think there's a few problems here. The database migration for this new `DeletedById` attribute is missing.

Shouldn't `DeletedById` be optional? Note, nullable context is also missing on this type.

For the relationship to be discoverable without needing explicit configuration, I think we need to add 

`public User? DeletedBy { get; set; } = null!;`

See https://learn.microsoft.com/en-us/ef/core/modeling/relationships/conventions